dishonored2.asl

```diff
@@ -45,6 +45,24 @@ state("Dishonored2", "1.9")
     uint canInteract : 0x288D608, 0x1F6EE8, 0x13B0, 0x0;
 }
 
+state("Dishonored2_x64ShippingRetail", "1.9")
```

Comment:
I think I'd prefer giving these different version identifiers so if you need to differentiate them in the code you can (since they're different addresses)

Reply:
I've given the ms build a unique version, re-tightened the check, confirmed it works, and pushed.

I'm not overly familiar with Github's pull requests, do I need to do anything further or have my changes been automatically rolled in and are now visible to you?